### DISCUSSION TITLE
Problem with Conform and Lua-Buffers in NvChad-2.5

### DISCUSSION THREAD

**Discussion by tevaughan** [NvChad/NvChad#2902] [Q&A] [answered]
I have conform working to format on save in NvChad-2.5. C++ seems properly to invoke clang-format according to my ~/.clang-format.
I have conform configured to use stylua for lua, but, while each lua buffer is indeed formatted on save, it is not formatted according to ~/.stylua.toml. This makes me think that conform is using lsp. stylua works correctly on the command-line. How do I debug and fix this problem?

**Comment by siduck** [comment, marked as answer]
delete .stylua.toml from ~/.config/nvim

**Comment by tevaughan** [comment]
Thanks!

(Now that I see ‘.stylua.toml’ there, it seems an obvious thing to do!)

Thomas E. Vaughan
